documentation/markdown/usage/new-cli-parameter.md

```diff
@@ -0,0 +1,94 @@
+# Adding a new CLI parameter to initialize your server
+
+Sometimes there might be a value in your server configuration that you do not want to hardcode.
+A good example of this is the password of your email server,
+should you have one enabled.
+Fortunately, due to the customizability of the server,
+it is possible to add such an option entirely through additions in your Components.js configuration.
+Below is an overview of the required additions.
+The example will cover how to use a variable for the email server password,
+but can be generalized to any situation.
```

Comment:
```suggestion
Sometimes, there is a value in your server configuration that you do not want to hardcode.
A good example of this is the password for your email account,
should you have one enabled.
Fortunately, due to the customizability of the server,
it is possible to add such an option entirely through additions to your `Components.js` configuration.
Below is an overview of the required additions.
The example covers how to use a variable for the email account password,
but it can be generalized to any situation.
```